locales/nb_NO/pcsx2_Main.po

```diff
@@ -34,21 +35,21 @@ msgstr "Parsingsfeil"
 
 #: common/include/Utilities/Exceptions.h:279
 msgid "Your machine's hardware is incapable of running PCSX2.  Sorry dood."
-msgstr "Din maskins maskinvare har ikke mulighet til å kjøre PCSX2. Beklager."
+msgstr "Maskinvaren er ikke i stand til å kjøre PCSX2.  Beklager dood."
 
 #: common/src/Utilities/Exceptions.cpp:204
 msgid "Oh noes! Out of memory!"
-msgstr "Oi! Da var det tomt for minne!"
+msgstr "Oi da! Da var det tomt for minne!"
 
 #: common/src/Utilities/Exceptions.cpp:219
 msgid ""
-"Virtual memory mapping failure!  Your system may have conflicting device "
-"drivers, services, or may simply have insufficient memory or resources to "
-"meet PCSX2's lofty needs."
+"Virtual memory mapping failure!  Your system may have conflicting device drivers, "
+"services, or may simply have insufficient memory or resources to meet PCSX2's lofty "
+"needs."
```

Comment:
Don't think these English strings require changes.

Reply:
I did not make those changes.